### PR TITLE
[ASM] - EXPANDR-6923 Prisma cloud changes

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-6923

## Description
Prisma cloud event search changes and fixing Azure enrichment bug.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml [hunk 1]
```diff
@@ -780,6 +781,458 @@ tasks:
...
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: df44266c-def7-40d4-88df-b6c4b716314b
+      iscommand: false
+      name: Event search in Prisma Cloud
+      type: title
+      version: -1
+      description: ''
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml:804]: Does this need a description?

Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml:804]: its a section header, so it doesn't

**Comment by johnnywilkes** [comment]
@capanw , if you can add the ticket to the related issue section, please https://jira-dc.paloaltonetworks.com/browse/EXPANDR-6923

**Review by johnnywilkes — APPROVED** [review]
@capanw , i approve

**Review by BigEasyJ — APPROVED** [review]

**Comment by capanw** [comment]
@YuvHayun, approved from our side. Please proceed to review and merge.

**Comment by YuvHayun** [comment]
hey @capanw @johnnywilkes, please pull from master and push changes.

**Comment by capanw** [comment]
Hello @YuvHayun, did a rebase with latest changes from master.

**Comment by YuvHayun** [comment]
Merging to internal pr.

**Review by YuvHayun — APPROVED** [review]

**Pull request merged**